src/extras/named-register.js

```diff
@@ -35,7 +35,7 @@
         firstNamedDefine = null;
       });
     }
-    return register.apply(this, arguments);
+    return register.apply(deps, declare);
```

Comment:
This should probably be:
```js
return register.apply(this, [deps, declare]);
```

I don't really understand why the unit tests doesn't pick up on this issue, but here's a repro with the above fix:

https://codesandbox.io/s/broken-sun-c3xjm?file=/index.html

See the `named-register-fixed.js`

Reply:
Ah good catch, I thought I had tested this but now that I see the problem I wonder why my testing wouldn't have noticed that it wasn't correct.

I've updated it now.